packages/shared/package.json

```diff
@@ -5,26 +5,26 @@
   "main": "index.js",
   "dependencies": {
     "@kiwicom/mobile-accessibility": "^0",
+    "@kiwicom/mobile-config": "^0",
     "@kiwicom/mobile-localization": "^0",
     "@kiwicom/mobile-navigation": "^0",
     "@kiwicom/mobile-orbit": "^0",
-    "@kiwicom/mobile-config": "^0",
     "@kiwicom/rnmodules": "^0",
     "@ptomasroos/react-native-multi-slider": "^1.0.0",
     "prop-types": "^15.6.2",
     "querystring": "^0.2.0",
     "react-native-image-progress": "^1.1.1",
-    "react-native-maps": "0.21.0",
+    "react-native-maps": "github:react-native-community/react-native-maps#6c225af612d4681a00d4849b60774d7c6426835e",
     "react-native-modal": "^7.0.0",
     "react-native-read-more-text": "^1.1.0"
   },
   "peerDependencies": {
+    "@kiwicom/universal-components": "^0",
```

Comment:
If we're using TextInput from universal, there were some changes in prop names in v0.0.16.

Reply:
We definitely need to get better at maintaining some kind of changelog with breaking changes, etc. I will create an issue for it.

Reply:
Ok, thanks for the heads up, I am still on 0.0.14, and we don't have any textInput I think

Reply:
https://github.com/kiwicom/margarita/issues/595